validators.py

```diff
@@ -1,7 +1,23 @@
-from pydantic import BaseModel, conlist, conint, Field
+from datetime import datetime
+
+from pydantic import BaseModel, validator
 
 
 class DatesAndPriceCheckInputValidator(BaseModel):
-    listing_ids: conlist(conint(gt=0), min_items=1) = Field(
-        title="Listing ids to check on"
-    )
+    listing_ids: list
+    desired_checkin: str
+
+    @validator('listing_ids')
+    def listing_must_greater_than_zero(cls, v):
```

Comment:
Compared to the previous method, your validator for `listing_ids` does not ensure that each element of the list is an integer which has a value > 0. I would recommend using the previous method in this case.

Reply:
Wanted to use this new Method as the error we were raising was too generic in the previous function. I shall add the new integer value > 0 as another condition. What do you think about this?